### PR TITLE
Added support to parse RTTI for GCC

### PR DESCRIPTION
**Pull request by r00tus3r**
#6851

### CODE REVIEW

File: libr/anal/rtti_itanium.c [hunk 1]
```diff
@@ -65,18 +163,62 @@ R_API void r_anal_rtti_itanium_print_class_type_info(RVTableContext *context, ut
...
+            si_class_type_info si_cti;
+            if (!rtti_itanium_read_si_class_type_info (context, colAddr, &si_cti)) {
+                    eprintf ("Failed to parse Type Info at 0x%08"PFMT64x" (referenced from 0x%08"PFMT64x")\n", colAddr, colRefAddr);
+                    return;
+            }
+            rtti_itanium_print_si_class_type_info (&si_cti, colAddr, "");
+        }
+
+        if (r_str_cmp(flag->name, class_type_info_name, r_str_len_utf8(flag->name))) {
+            //r_cons_println (flag->name);
```
Comment (radare) [inline libr/anal/rtti_itanium.c]: use tabs

File: libr/anal/rtti_itanium.c [hunk 2]
```diff
@@ -65,18 +163,62 @@ R_API void r_anal_rtti_itanium_print_class_type_info(RVTableContext *context, ut
...
+            si_class_type_info si_cti;
+            if (!rtti_itanium_read_si_class_type_info (context, colAddr, &si_cti)) {
+                    eprintf ("Failed to parse Type Info at 0x%08"PFMT64x" (referenced from 0x%08"PFMT64x")\n", colAddr, colRefAddr);
+                    return;
+            }
+            rtti_itanium_print_si_class_type_info (&si_cti, colAddr, "");
+        }
+
+        if (r_str_cmp(flag->name, class_type_info_name, r_str_len_utf8(flag->name))) {
```
Comment (radare) [inline libr/anal/rtti_itanium.c]: space before (

Comment (radare) [reply libr/anal/rtti_itanium.c]: there are many more places with no space before (

File: libr/anal/rtti_itanium.c [hunk 3]
```diff
@@ -42,16 +48,108 @@ typedef struct vmi_class_type_info_t {
...
+                                "%s\tCount of base classes: %d" "\n",
+				prefix, addr,
+                                prefix, vmi_cti->vtable_addr,
+                                prefix, vmi_cti->name_addr,
+                                prefix, vmi_cti->vmi_flags,
+                                prefix, vmi_cti->vmi_base_count);
+
+	int i;
+        for(i = 0; i < vmi_cti->vmi_base_count; i++) {
+                r_cons_printf("%s\t\tBase class type descriptor address: 0x%08"PFMT32x "\n"
```
Comment (radare) [inline libr/anal/rtti_itanium.c]: dont indent with spaces

File: libr/anal/rtti_itanium.c [hunk 4]
```diff
@@ -42,16 +48,108 @@ typedef struct vmi_class_type_info_t {
...
+                                "%s\tReference to RTTI's type class: 0x%08"PFMT32x "\n"
+                                "%s\tReference to type's name: 0x%08"PFMT32x "\n",
+				prefix, addr,
+                                prefix, cti->vtable_addr,
+                                prefix, cti->name_addr);
+}
+
+static void rtti_itanium_print_vmi_class_type_info (vmi_class_type_info *vmi_cti, ut64 addr, const char *prefix) {
+        r_cons_printf ("%sType Info at 0x%08"PFMT64x ":\n"
+                                "%s\tReference to RTTI's type class: 0x%08"PFMT32x "\n"
```
Comment (radare) [inline libr/anal/rtti_itanium.c]: dont use \t in any output of r2 just use 2 spaces or so

**Comment by r00tus3r** [comment]
Okay...i'll make the changes and give a PR

**Pull request merged**